pom.xml

```diff
@@ -136,7 +136,8 @@
                 <module>oxford-url-shortener</module>
                 <module>proxy</module>
                 <module>root-webapp</module>
-                <module>sirlouie</module>
+                <!-- Disabled in June 2021 in WL-5346 -->
+                <!--<module>sirlouie</module>-->
```

Comment:
I think we want to remove the code and the supporting code from citations from the build so that it's not confusing about if it's in use any more.

Reply:
Thanks for the feedback @buckett, would you mind reviewing I'm in the right path?